src/airflow/airqo_etl_utils/airnow_api.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+import requests
+
+from airqo_etl_utils.config import configuration
+
+
+class AirNowApi:
+    def __init__(self):
+        self.AIRNOW_BASE_URL = configuration.AIRNOW_BASE_URL
+        self.AIRNOW_API_KEY = configuration.AIRNOW_API_KEY
+        self.AIRNOW_METADATA_JSON_FILE = configuration.AIRNOW_METADATA_JSON_FILE
+
+    def get_metadata(self):
+        with open(self.AIRNOW_METADATA_JSON_FILE) as file:
+            devices = json.load(file)
+
+        return list(devices)
+
```

Comment:
Mocking BAM metadata from a json file. To be replaced with API call to device registry once the BAM data is available.